proto/viam/service/slam/v1/slam.proto

```diff
@@ -26,6 +26,29 @@ service SLAMService {
       get: "/viam/api/v1/service/slam/{name}/map"
     };
   }
+
+  // GetPositionNew returns the current estimated position of the robot with
+  // respect to a returned reference frame
```

Comment:
This should say "component reference" instead of "reference frame" (explicitly excluding the frame system).